.config/dictionary.txt

```diff
@@ -229,3 +229,114 @@ robertdebock
 ssbarnea
 willthames
 yatesr
+getmatches
+matchlines
+matchtasks
+matchdir
+codenotify
+sourcegraph
+filetree
+matchvar
+varstring
+lintables
+caplog
+autodiscovery
+releasenotes
+mockreturn
+dotconfig
+formatstr
+placefolder
+blockinfile
+representer
+Representer
+scalarint
+typehint
+fullwidth
+typehints
+keepends
+matcherror
+formsyntax
+placefolder
+listrules
+listtags
+getmatches
+listtags
+testrun
+autodetected
+commandline
+ruledirs
+htmldir
+formetting
+filelock
+RULEDIRS
+placefolder
+playholder
+buildset
+apport
+bracketsmatchtestfile
+bracketsmatchtest
+netcommon
+dellemc
+redirections
+blockincludes
+nomatchestest
+templatevars
+simpletask
+taskshandlers
+taskhandler
+rolepath
+subresults
+Autobuild
+autobuild
+regexes
+argspecs
+insertafter
+chunksize
+sortfunc
+CODENOTIFY
+createfile
+validable
+Sshell
+levelname
+becomeuserabove
+nitpicky
+linkcheck
+autoclass
+opensearch
+modindex
+pageview
+moduleauthor
+sectionauthor
+dharmabumstead
+intersphinx
+extlinks
+apidoc
+viewcode
+programoutput
+sphinxcontrib
+pickleable
+slaveinput
+addoption
+pluginmanager
+filesspot
+roundrip
```

Comment:
roundrip is clearly a typo

Reply:
Thanks for pointing it out, I'll change it.